dojo/importers/reimporter/reimporter.py

```diff
@@ -83,9 +83,9 @@ def process_parsed_findings(self, test, parsed_findings, scan_type, user, active
                 if finding.false_p or finding.out_of_scope or finding.risk_accepted:
                     logger.debug('%i: skipping existing finding (it is marked as false positive:%s and/or out of scope:%s or is a risk accepted:%s): %i:%s:%s:%s', i, finding.false_p, finding.out_of_scope, finding.risk_accepted, finding.id, finding, finding.component_name, finding.component_version)
                 elif finding.mitigated or finding.is_mitigated:
-                    logger.debug("item mitigated time: " + str(item.mitigated.timestamp()))
-                    logger.debug("finding mitigated time: " + str(finding.mitigated.timestamp()))
                     if item.mitigated:
+                        logger.debug("item mitigated time: " + str(item.mitigated.timestamp()))
+                        logger.debug("finding mitigated time: " + str(finding.mitigated.timestamp()))
```

Comment:
Wouldn't we have the same problem here, if `item` is mitigated but `finding` isn't? Maybe line 85 needs an `and` instead of an `or`.

Reply:
I agree, the original code was with the `or` condition but I didn't really understand why, honestly the whole block that reactivates the finding should be gone, and have much more conditions imo
```python
                    logger.debug('%i: reactivating: %i:%s:%s:%s', i, finding.id, finding, finding.component_name, finding.component_version)
                    finding.mitigated = None
                    finding.is_mitigated = False
                    finding.mitigated_by = None
                    finding.active = True
                    finding.verified = verified
```